assets/src/settings-page/index.js

```diff
@@ -94,8 +95,13 @@ ErrorNotice.propTypes = {
 
 /**
  * Settings page application root.
+ *
+ * @param {Object} props Component props.
+ * @param {string} props.focusedSection The initially focused focusedSection.
  */
 function Root() {
+	const [ focusedSection, setFocusedSection ] = useState( global.location.hash.replace( /^#/, '' ) );
```

Comment:
`setFocusedSection` isn't yet used. It will be set in a `hashchange` event handler?

Reply:
Yup, in 754aadd8572de9744af0f99953b077c3bded9bf9